### DISCUSSION TITLE
move course title to a config option?

### DISCUSSION THREAD

**Discussion by Alex-Jordan** [openwebwork/webwork2#2051] [Ideas]
A course's title (like "Calculus 1") is stored in the database. But does it make sense to store it there? Is it somehow valuable for analyzing courses that share the same title?

Currently since it's a database value, the only tool to edit that title is in the admin course. If this were a config value instead, instructors could easily set (or change) their own course title with a new field in Course Config. We could seed `simple.conf` with this title when a new course is created.

**Comment by drgrice1** [comment]
There is not real reason for it not to be in the database.  Ideally there would be no such thing as `simple.conf` or even `course.conf`, and everything would be in the database.  That is how it will be with webwork3.  There is no reason that it could not be made so that the instructor could edit the title even though it is in the database.

**Comment by taniwallach** [comment]
I would be a bit wary of the instructor being able to change the course title.
A site administrator might want a course title to include "extra" details such a semester / group number / institution, which a typical instructor might be happy to delete.
At least on the login page - such extra details are quite important, as the "magic" inside the course_id to differentiate groups/semesters can be gibberish to regular people, in particular due to the length limits on course ids.
Maybe it would be better to keep a "full detail" title and a "short" title, and give the instructors access to edit the short title.
